db/default/users.rb

```diff
@@ -62,7 +62,7 @@ def create_admin_user
       role = Spree::Role.find_or_create_by(name: 'admin')
       admin.spree_roles << role
       admin.save
-      say "Done!"
+      say "New admin user persisted!"
```

Comment:
I changed this because when running `script/setup` this line is misleading (when it prints "Done!" to console it looks as if the entire setup process is complete when in fact it is not)